.github/workflows/create-release-branch.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     permissions:
       contents: write
     outputs:
-      branch-name: ${{ steps.create-branch.branch_name }}
+      branch-name: ${{ steps.create-branch.outputs.branch_name }}
```

Comment:
The typo was causing the later tests to be run against dev instead of the just-created release branch. This should fix that.